module/utils.mjs

```diff
@@ -1061,18 +1070,38 @@ export function getHumanReadableAttributeLabel(attr, { actor }={}) {
   }
 
   // Check if the attribute is already in cache.
-  let label = _attributeLabelCache.get(attr);
+  let label = item ? null : _attributeLabelCache.actor.get(attr);
   if ( label ) return label;
+  let name;
+  let type = "actor";
 
-  // Derived fields.
-  if ( attr === "attributes.init.total" ) label = "DND5E.InitiativeBonus";
-  else if ( (attr === "attributes.ac.value") || (attr === "attributes.ac.flat") ) label = "DND5E.ArmorClass";
-  else if ( attr === "attributes.spelldc" ) label = "DND5E.SpellDC";
+  const getSchemaLabel = (attr, type) => {
```

Comment:
This wasn't part of the original code, but since we sometimes have a full Actor or Item instance, we can actually look up the correct schema, rather than iterating over all of the,.